signer/xpi/x509.go

```diff
@@ -61,33 +62,63 @@ func (s *XPISigner) monitorRsaCacheSize() {
 	}
 }
 
-// retrieve a key from the cache or generate one if it takes too long
-// or if the size is wrong
+type rsaKey struct {
+	lock  sync.Mutex
+	age   time.Time
```

Comment:
nit: I think we should rename this to `creationTime` or `createdAt` since age implies a duration (like the other age vars) to me